dkist/io/asdf/converters/tiled_dataset.py

```diff
@@ -30,6 +32,12 @@ def to_yaml_tree(cls, tiled_dataset, tag, ctx):
         # If the history key has been injected into the meta, do not save it
         meta.pop("history", None)
         tree["meta"] = meta
+        # Go into dataset header attributes and replace with {"offset": ..., "size": ...}
```

Comment:
heh, I just realised doing this will affect what's in memory:
```python
ds = TiledDataset(...)
asdf.write(ds)
ds[0,0].headers == {"offset": ...}
```
which isn't ideal.

I think this might mean it's custom tag for us.